### PR TITLE
pdftk and qpdf

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

File: pdftk.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.com/pdftk-java/pdftk
+      tag: v${{package.version}}
+      expected-commit: e4292c8f1bd2580a44d3cbf3570a4505bd3a74b6
+  
+  - name: Build
+    runs: |
+      gradle -Dorg.gradle.daemon=false buil
```
Comment (pdeslaur) [inline pdftk.yaml:32]: Is 'buil' a typo?

File: qpdf.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            - samurai
+            - zlib-dev
+pipeline:
+    - uses: fetch
+      with:
+        expected-sha256: d9321f5fbc50251803630a5604ddc5ed9a4d93bc023d9a7436a302e7c9741259
+        uri: https://github.com/qpdf/qpdf/releases/download/v${{package.version}}/qpdf-${{package.version}}.tar.gz
+    - name: Build
+      runs: |
+        CXXFLAGS="$CXXFLAGS" \
```
Comment (pdeslaur) [inline qpdf.yaml:30]: Are these 2 lines needed?

File: pdftk.yaml (hunk 1 above)
Comment (developer-guy) [reply pdftk.yaml:32]: ah yes! thank you

File: qpdf.yaml (hunk 2 above)
Comment (developer-guy) [reply qpdf.yaml]: nope I removed one additional parameter from them so its ok to remove them

File: pdftk.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.com/pdftk-java/pdftk
+      tag: v${{package.version}}
+      expected-commit: e4292c8f1bd2580a44d3cbf3570a4505bd3a74b6
+
+  - name: Build
+    runs: |
+      gradle -Dorg.gradle.daemon=false build
+      ls -latr build/libs
```
Comment (pdeslaur) [inline pdftk.yaml:34]: This `ls` is probably not needed!

**Review by pdeslaur — DISMISSED** [review]
Nice!

**Review by pdeslaur — APPROVED** [review]

**Pull request merged**